mojblocks.php

```diff
@@ -388,6 +388,30 @@ function mojblocks_register_blocks()
         ]
     );
 
+    register_block_type(
+        'mojblocks/auto-item-list',
```

Comment:
the naming of the block in frontend 'most recent items' is diff then here 'auto item list'  .. ideally these should be different but i feel like the name of the block may change again.

Reply:
Near enough